airflow-core/src/airflow/assets/manager.py

```diff
@@ -54,6 +55,63 @@
 log = structlog.get_logger(__name__)
 
 
+@contextmanager
+def _acquire_apdr_lock(
+    *,
+    session: Session,
+    dag_id: str,
+    partition_key: str,
```

Comment:
it seems the params, dag_id and partition_key are unused

Reply:
yep, removed. Thanks!